package/skelcd-control-leanos.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jul 20 12:42:50 UTC 2021 - Stefan Weiberg <sweiberg@suse.com>
+
+- Adapt the product to SLE-15-SP4
```

Comment:
Please add a bugzilla number for reference, otherwise our Jenkins would later reject to submit the change.

Reply:
opened a bug and added the reference